src/Compilers/VisualBasic/Portable/Parser/ParseConditional.vb

```diff
@@ -248,7 +251,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
         Private Function ParseRegionDirective(hashToken As PunctuationSyntax) As RegionDirectiveTriviaSyntax
             Debug.Assert(CurrentToken.Kind = SyntaxKind.IdentifierToken AndAlso DirectCast(CurrentToken, IdentifierTokenSyntax).PossibleKeywordKind = SyntaxKind.RegionKeyword,
-                         "ParseRegionDirective called with wrong token.")
+                         $"{NameOf(ParseRegionDirective)} called with wrong token.")
```

Comment:
This (and other similar places) could probably be a simple string concatenation rather than an interpolation.